build.gradle

```diff
@@ -0,0 +1,122 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java project to get you started.
+ * For more details take a look at the Java Quickstart chapter in the Gradle
+ * User Manual available at https://docs.gradle.org/5.2.1/userguide/tutorial_java_projects.html
+ */
+
+plugins {
+    // Apply the java plugin to add support for Java
+    id 'java'
+    // git clone, etc. 
+    id "org.ajoberstar.grgit" version "2.3.0"       
+}
+
+sourceCompatibility = '1.8'
+targetCompatibility = '1.8'
+version = '0.1.0'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    // Use JUnit test framework
+    testCompile "junit:junit-dep:4.11"
+    // com.googlecode.json.simple
+    compile "com.googlecode.json-simple:json-simple:1.1.1"
+    // org.apache.commons.math3
+    compile "org.apache.commons:commons-math3:3.6.1"
+    // gov.nist.math.jama
+    compile "gov.nist.math:jama:1.0.2"
+}
+
+// method to add/update an external project from git and add it to the 
+// sourceset
+ext.addExternalProject = { path, url, srcDir ->
+  File projDir = new File("$path")
+  // clone project if it doesn't exist locally
+  if(!projDir.exists()) {
+    grgit.clone(dir: projDir, uri: url)
+  }
+  // pull the project if it does exist locally
+  else {
+    grgit.open(dir: projDir)
+    grgit.pull()
+    grgit.close()
+  }
+
+  // add to sourceset
+  sourceSets {
+    main {
+      java {
+          srcDirs "$path/$srcDir"
+      }
+    }
+  }
+}
+
+// clone/update external libs from github, at least until they're available in
+// maven
+task cloneExternalProjects {
+  description 'Clones/Updates external project(s).'
+  
+  doFirst {
+    // neic-traveltime
+    addExternalProject("$projectDir/lib/neic-traveltime", 
+      "https://github.com/usgs/neic-traveltime.git", "src")
+
+    // earthquake-processing-formats
+    addExternalProject("$projectDir/lib/earthquake-processing-formats", 
+      "https://github.com/usgs/earthquake-processing-formats.git", "src")
+  }
+}
+
+// copy required model files
+task copyModelFiles(type: Copy) {
+  description 'Copies required model files to build directory.'
+  dependsOn 'cloneExternalProjects'
+  from "models", "$projectDir/lib/neic-traveltime/models/"
+  exclude "README.txt"
+  into "$buildDir/models"
+}
+
+// make sure we always update the external projects and copy models prior to 
+// complile
+defaultTasks 'cloneExternalProjects', 'copyModelFiles'
+compileJava.dependsOn 'cloneExternalProjects', 'copyModelFiles'
+
+// for now, just add traveltime and processing formats to the source set,
```

Comment:
I think this is no longer needed, since files were moved to the default location